cwl_runner/dx_cwl_runner/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.0"
+__version__ = "0.2.0"
```

Comment:
Changed with the version of cwl_runner.

Note that in dxcint, this file exists as well (dxcint/dxcint/__init__.py). This file has version 0.1.0,  but dxcint never had this version (it had 0.0.2, not it is 0.0.3). Do I misunderstand what these files refer to, or was it set incorrectly? @aogorodnikov-dna

Reply:
Also this file was changed after tests ran successfully, I will rerun them once review is finished - in case there are more changes. I don't expect this to affect anything.